packages/maskbook/src/plugins/Collectible/UI/OrderRow.tsx

```diff
@@ -137,7 +137,7 @@ export function OrderRow({ order, isDifferenceToken, acceptable }: IRowProps) {
                     <TableCell>
                         <Typography className={classes.content}>
                             {order.expirationTime &&
-                                !new BigNumber(order.expirationTime).isZero() &&
+                                !isZero(order.expirationTime) &&
```

Comment:
I think no need to operate `expirationTime` with `BigNumber`(Maybe change it in another PR)

Reply:
open new pr change this logic?